### ISSUE TITLE
Clean up property situation in NetworkPlan

### ISSUE THREAD

**Issue by vr2262** [SEL-Columbia/Sequencer#63]
See https://github.com/SEL-Columbia/Sequencer/pull/62#discussion_r61439140

**Comment by vr2262** [comment]
@chrisnatali nice branch for this right here: https://github.com/SEL-Columbia/Sequencer/compare/vr2262-issue-63-properties

Are you going to make a release branch or should i make a PR into master?

**Comment by vr2262** [comment]
@chrisnatali

**Comment by chrisnatali** [comment]
The point is that properties are not justified here?

**Comment by vr2262** [comment]
Yes, they don't get you anything (and you can always go back if you want).

**Comment by chrisnatali** [comment]
Good.  PR is fine, though I don't see this as a priority.  We can release this if another urgent fix comes up.  Otherwise, I think more broad issues like #64 & #65 should be the focus now.
